### ISSUE TITLE
RxValueAnimator support for RxJava 1.2+

### ISSUE THREAD

**Issue by laszlo-galosi** [0ximDigital/RxAnimations#2]
Recently, RxJava 1.2.0 has been released and they've made some backward-incompatible changes related to Completables, specifically:

- CompletableSubscriber -> rx.CompletableSubscriber
- CompletableOnSubscribe -> rx.Completable.OnSubscribe

After upgrading to it, I'm getting the following gradle build issues on :
```
RxValueAnimator.from(myValueAnimator, animator ->{
              ...
 });
```
`Error:(347, 24) error: cannot access CompletableOnSubscribe class file for rx.Completable$CompletableOnSubscribe not found`

**Comment by 0ximDigital** [comment]
Hi, thank you for reporting this issue.

I will look into it and publish fix soon, so stay tuned :)

**Comment by laszlo-galosi** [comment]
Also look into RxAnimationBuilder which has the same issue too.

**Comment by 0ximDigital** [comment]
Fixed in release version 0.9.0.

Once again, thank you for reporting this issue 👍
